### PR TITLE
v1.4 Minor fix and update PPP, UG & DG

### PR DESCRIPTION
**Pull request by yican95**

### CODE REVIEW

File: docs/team/yican95.adoc [hunk 1]
```diff
@@ -20,47 +21,67 @@ approximately 30 KLOC.
 
 == Summary of contributions
```
Comment (AZhiKai) [inline docs/team/yican95.adoc]: Update it to

```suggestion
== Summary of Contributions
```

**Comment by AZhiKai** [comment]
Hi @yican95  i see that you did some code changes.

Do give us a brief explanation on why, so we're aware if there's any logic changes.

**Comment by yican95** [comment]
I change the message usage and constraints for menu commands.
I fix some minor bug for some menu command not catch exception

**Comment by AZhiKai** [comment]
> I change the message usage and constraints for menu commands.
> I fix some minor bug for some menu command not catching exception

Sounds good!

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -2019,3 +2000,94 @@ These instructions only provide a starting point for testers to work on; testers
...
+
+.. Test case: `sort-menu name` +
+   *Expected*: Sorts the menu by name. Sorting method shown in status message.
+
+. Sorting the menu by price
+
+.. Test case: `sort-menu price` +
+   *Expected*: Sorts the menu by price. Sorting method shown in status message.
+
+. Giving a discount to all item
```
Comment (m4dkip) [inline docs/DeveloperGuide.adoc]: Change to "...discount to all _items_"

File: docs/DeveloperGuide.adoc [hunk 3]
```diff
@@ -2019,3 +2000,94 @@ These instructions only provide a starting point for testers to work on; testers
...
+
+.. Prerequisites: You must have at least 1 item in the menu.
+.. Test case: `discount-item ALL dp/20` +
+   *Expected*: Gives all items in the menu a discount. Number of item given a discount shown in status message.
```
Comment (m4dkip) [inline docs/DeveloperGuide.adoc]: Change to "...Number of _items_ given a discount..."

File: docs/DeveloperGuide.adoc [hunk 4]
```diff
@@ -2019,3 +2000,94 @@ These instructions only provide a starting point for testers to work on; testers
...
+.. Incorrect commands to try: `discount-item -2 dp/20`, `discount-item string dp/20`
+
+. Adding recipe to an item
```
Comment (m4dkip) [inline docs/DeveloperGuide.adoc]: Change to "Adding _a_ recipe to an item"

**Pull request merged**